packages/solidity-contracts/deploy/mainnet/010.gateway_rate_limit_usdc.ts

```diff
@@ -0,0 +1,28 @@
+import type { HardhatRuntimeEnvironment } from 'hardhat/types';
+import type { DeployFunction } from 'hardhat-deploy/dist/types';
+import { parseUnits } from 'ethers';
+
+const USDC_ADDRESS = '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48';
+const ONE_WEEK_IN_SECONDS = 3600 * 24 * 7;
```

Comment:
can we move these to the constants file? they're being used in multiple scripts.